Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,45 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            const string name = "Hancock";
+            const string surname = "Johnson";
+            const int age = 20;
+            const float weight = 81;
+            const float heightcm = 180;
+            const float heightm = heightcm / 100;
+            float heightsq = heightm * heightm;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'heightsq'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836333)